chpl/chpl-service/src/main/java/gov/healthit/chpl/service/CertificationCriterionService.java

```diff
@@ -57,6 +60,11 @@ public CertificationCriterion get(String certificationCriterionDescriptor) {
         }
     }
 
+    public List<CertificationCriterion> getByNumber(String certificationCriterionNumber) {
+        return criteriaByNumberMap.containsKey(certificationCriterionNumber)
+                ? criteriaByNumberMap.get(certificationCriterionNumber) : null;
```

Comment:
I think this can just be a `get`

From the Java docs:

> get(Object key)
> Returns the value to which the specified key is mapped, or null if this map contains no mapping for the key.

Reply:
Agree